src/main/car/functionality/DealershipUI

```diff
@@ -0,0 +1,131 @@
+		JLabel Renting = new JLabel("Renting: ");
+		JLabel Acquisition = new JLabel("Vehicle Acquisition: ");
+		JLabel Transfer = new JLabel("Transfer Inventory to: ");
+		JLabel Export = new JLabel("Export: ");
+		JTextField enterName = new JTextField("Dealer Name here");
+		JTextField transferInventory = new JTextField("Receiving Dealer ID here");
+		
+		JButton changeName = new JButton("save");
+		changeName.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton disableRent = new JButton("disable");
+		disableRent.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton enableRent = new JButton("enable");
+		enableRent.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton disableAcquisition = new JButton("disable");
+		disableAcquisition.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton enableAcquisition = new JButton("enable");
+		enableAcquisition.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton transferAllVehicles = new JButton("transfer");
+		transferAllVehicles.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton XMLExport = new JButton("XML");
+		XMLExport.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton JSONExport = new JButton("JSON");
+		JSONExport.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		JButton specificVehicleScreen = new JButton("Specific Vehicle");
+		specificVehicleScreen.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				
+			}
+		});
+		
+		dealerDetails.add(ID);
+		
+		Options.add(Name);
+		Options.add(enterName);
+		Options.add(changeName);
+		
+		Options.add(Renting);
+		Options.add(enableRent);
+		Options.add(disableRent);
+		
+		Options.add(Acquisition);
+		Options.add(enableAcquisition);
+		Options.add(disableAcquisition);
+		
+		Options.add(Transfer);
+		Options.add(transferInventory);
+		Options.add(transferAllVehicles);
+		
+		Options.add(Export);
+		Options.add(XMLExport);
+		Options.add(JSONExport);
+		
+		Options.setBorder(BorderFactory.createTitledBorder(BorderFactory.createEtchedBorder(), "Dealer Details"));
+		
+		specificVehicle.add(specificVehicleScreen);
+		
+		dealerDetails.add(Options);
+		dealerDetails.add(specificVehicle);
+		
+		dealerDetails.pack();
+		dealerDetails.setVisible(false);
+
+
+		//dealer search screen
+		JFrame dealerSearch = new JFrame("Search for a dealer");
+	    JLabel dealerOfInterest = new JLabel("Dealer: ");
+	    JTextField searchTerm = new JTextField("enter dealer name here...");
```

Comment:
the dealer search should look for a dealer id not name since name is not unique. You would find the dealer doing dealerGroup.getDealer(id)

Reply:
I'll make adjustments to clear up the ambiguity

Reply:
Addressed in line 135 after adding parameter to work with a DealerGroup